webpack.config.js

```diff
@@ -74,7 +74,7 @@ const config = (module.exports = {
   // output to "dist"
   output: {
     path: BUILD_PATH + "/app/dist",
-    filename: "[name].[contenthash].js",
+    filename: devMode ? "[name].bundle.js" : "[name].[contenthash].js",
```

Comment:
using any kind of hash in dev mode is not recommended https://github.com/webpack/webpack/issues/10796#issuecomment-620844831